src/reference/90-Developers-Guide/06-Datatype/00.md

```diff
@@ -5,12 +5,12 @@ out: Datatype.html
 Datatype
 --------
 
-sbt-datatype is a code generation library and an sbt autoplugin that generates
+`sbt-datatype` is a code generation library and an sbt autoplugin that generates
```

Comment:
I'm totally open to adding a style guide. *sbt* is always lower cased, and not monospaced when it refers to the project or the build tool in general. It is monospaced when it's referring to the shell script `sbt` that gets install on one's machine.